<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>640</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>851</width>
     <height>601</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Drill Down</string>
    </attribute>
    <widget class="QListWidget" name="stateListWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>141</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>56</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>State</string>
     </property>
    </widget>
    <widget class="QLabel" name="stateStatus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>State</string>
     </property>
    </widget>
    <widget class="QListWidget" name="countyListWidget">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>161</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="districtListWidget">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>60</y>
       <width>181</width>
       <height>192</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
    <widget class="QListWidget" name="schoolListWidget">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>60</y>
       <width>301</width>
       <height>192</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>56</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>County</string>
     </property>
    </widget>
    <widget class="QLabel" name="countyStatus">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>County</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>20</y>
       <width>56</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>District</string>
     </property>
    </widget>
    <widget class="QLabel" name="districtStatus">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>District</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>20</y>
       <width>56</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>School</string>
     </property>
    </widget>
    <widget class="QLabel" name="schoolStatus">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>30</y>
       <width>301</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>School</string>
     </property>
    </widget>
    <widget class="QPushButton" name="schoolNext">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>260</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="countyPageStatus">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>260</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>n/a</string>
     </property>
    </widget>
    <widget class="QLabel" name="schoolPageStatus">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>260</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>n/a</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stateNext">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>260</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="countyNext">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>260</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="countyPrev">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>260</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="statePageStatus">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>260</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>n/a</string>
     </property>
    </widget>
    <widget class="QPushButton" name="districtPrev">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>260</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="districtNext">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>260</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="districtPageStatus">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>260</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>n/a</string>
     </property>
    </widget>
    <widget class="QPushButton" name="statePrev">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="schoolPrev">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>260</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Geometry</string>
     </property>
    </widget>
    <widget class="QLabel" name="geometryStatus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>State</string>
     </property>
    </widget>
    <widget class="QPushButton" name="geometryNext">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>520</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="geometryPageStatus">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>520</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>n/a</string>
     </property>
    </widget>
    <widget class="QPushButton" name="geometryPrev">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>520</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
    <widget class="QListWidget" name="geometryListWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>821</width>
       <height>181</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="checkoutGeometry">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>510</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Checkout Copy of Selected Geometry</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Checked-out Geometry</string>
    </attribute>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>811</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>351</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Geometries Checked to your Account</string>
     </property>
    </widget>
    <widget class="QLabel" name="stateStatus_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>581</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>These Boundaries load into QGIS for editing. Come back to this screen to save changes to your working copies</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
